lychee-bin/src/formatters/log.rs

```diff
@@ -0,0 +1,53 @@
+use log::Level;
+use std::io::Write;
+
+use crate::{formatters, options::ResponseFormat, verbosity::Verbosity};
+
+/// Initialize the logging system with the given verbosity level
+pub(crate) fn init_logging(verbose: &Verbosity, mode: &ResponseFormat) {
+    let mut builder = env_logger::Builder::new();
+
+    builder
+        .format_timestamp(None) // Disable timestamps
+        .format_module_path(false) // Disable module path to reduce clutter
+        .format_target(false) // Disable target
+        .filter_module("lychee", verbose.log_level_filter()) // Re-add module filtering
+        .filter_module("lychee_lib", verbose.log_level_filter()); // Re-add module filtering
+
+    // Enable color unless the user has disabled it
```

Comment:
Good idea. I've updated the comment and refactored the code slightly. For now, I'll keep the code inline here, because it's sort of "special" to this single use-case. However, I aligned the width with the response format now through the `TOTAL_RESPONSE_OUTPUT_WIDTH` constant.